### PR TITLE
Auth part2 - use auth

### PR DESCRIPTION
**Pull request by Khoacannotcode**
## Related Tickets:
- [#45195](https://edu-redmine.sun-asterisk.vn/issues/45195)

## WHAT this PR does?
- Use Auth in admin controllers
- Modify web.php
- Add viewProfile page

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -20,43 +12,149 @@
 | contains the 'web' middleware group. Now create something great!
 |
 */
+Route::get('lang/{lang}', [
+    'as' => 'lang.switch',
+    'uses' => 'LanguageController@switchLang']);
```
Comment (phuongha47) [inline routes/web.php]: ngoặc xuống hàng, chấm nè

Comment (Khoacannotcode) [reply routes/web.php]: ok

**Comment by phuongha47** [comment]
ready

**Comment by Khoacannotcode** [comment]
ready

**Pull request merged**